udata/core/dataset/api_fields.py

```diff
@@ -282,9 +282,28 @@
     'label': fields.String(description='The resource type display name')
 })
 
-
+# follow the specification of https://schema.data.gouv.fr/schemas/schemas.json
 catalog_schema_fields = api.model('CatalogSchema', {
-    'id': fields.String(description='The schema identifier'),
-    'label': fields.String(description='The schema display name'),
-    'versions': fields.List(fields.String, description='The available versions of the schema'),
+    'name': fields.String(),
+    'title': fields.String(),
+    'description': fields.String(),
+    'schema_url': fields.String(description="Often the link to the latest version"),
+    'schema_type': fields.String(enum=['tableschema', 'datapackage', 'jsonschema', 'other']),
+    'contact': fields.String(),
+    'examples': fields.List(fields.Nested(api.model('CatalogSchema.example', {
```

Comment:
I think we usually use PascalCase in models name, right? 
```suggestion
    'examples': fields.List(fields.Nested(api.model('CatalogSchemaExample', {
```

Reply:
I wanted to be more explicit that it's inner models (and not used outside the parent). But I can change it if you want

Reply:
Changed in ce72578f0 :-)